### DISCUSSION TITLE
Input Multiple Images in C#

### DISCUSSION THREAD

**Discussion by TamirHCL** [microsoft/onnxruntime-genai#835] [New features / APIs]
Seems that currently you can't input multiple images when using ONNX C# Runtime. 
Lack of this makes use of Phi 3.5 Vision kind of limited. 
Would it be possible prioritize this and add support for this now that Phi 3.5 Vision is almost meant for this?

**Comment by natke** [comment]
Hi @TamirHCL , support for multiframe in Phi 3.5 vision is in progress

**Comment by kunal-vaishnavi** [comment]
The new [Phi-3 vision](https://huggingface.co/microsoft/Phi-3-vision-128k-instruct-onnx) and [Phi-3.5 vision](https://huggingface.co/microsoft/Phi-3.5-vision-instruct-onnx) ONNX models have now been released. The new models support no-image, single-image, and multi-image scenarios.
